src/main/java/com/brandwatch/internship/fetchandstoretweets/entities/Mention.java

```diff
@@ -33,9 +52,22 @@ public Mention(long id, String idStr, String text, Date createdAt, String fromUs
         this.fromUserId = fromUserId;
         this.languageCode = languageCode;
         this.source = source;
-        this.queryId = queryId;
+
     }
 
+    private static Mention tweetToMention(Tweet tweet, long queryId){
```

Comment:
I'd tend to say these should be in a dedicated service, or a utils class of some kind, that knows how to convert this. There's no need the Mention needs to contain any logic, it doesn't even need to know what a `Tweet` is. There is not one truth, however, keeping domain objects and logic separate tends to be a good decision.